api/main.py

```diff
@@ -55,6 +58,13 @@ def main(request):
                 if env in SWARM:
                     SWARM[env] = ticket['key']
 
+    for name in SWARM:
```

Comment:
You could potentially also limit that iteration only to the test instances that still have a value of `1`.